packages/core/lib/utils.ts

```diff
@@ -1,5 +1,5 @@
 import { SPRITESHEET_NAMESPACE } from "./constants";
-import { Props, Optimize } from "./Props";
+import { type Props, type Optimize } from "./Props";
```

Comment:
```suggestion
import type { Props, Optimize } from "./Props";
```

Reply:
> 

While building process (No SSR, local build), the two warnings are gone :
```
node_modules/astro-icon/lib/utils.ts (2:9) "Props" is not exported by "node_modules/astro-icon/lib/Props.ts", imported by "node_modules/astro-icon/lib/utils.ts".
node_modules/astro-icon/lib/utils.ts (2:16) "Optimize" is not exported by "node_modules/astro-icon/lib/Props.ts", imported by "node_modules/astro-icon/lib/utils.ts". 
```

Reply:
If it's gone, then it's ok not?

Reply:
Sorry if i was not clear, i mean your solution is working fine for me as i mentioned.